_acs-aem-commons/features/contextual-content-variables/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: acs-aem-commons_feature
+title: Contextual Content Variables (CCVAR)
+description: Use properties directly in authored text
+redirect_from: /acs-aem-commons/features/contextual-content-variables.html
+date: 2021-05-21
+feature-tags: authoring
+initial-release: 5.0.5
+---
+
+## Purpose
+
+Contextual Content Variables is a tool built to enable authors to use content variables directly in their authored text.
+The tool replaces variables present in the content (either JSON or HTML) at render-time, keeping caching enabled for the
```

Comment:
Please add if server-side or client-side

Reply:
@kwin I've updated the description to include the fact that it's done on the server-side. thanks for calling it out